rasa/core/run.py

```diff
@@ -34,6 +34,12 @@ def create_http_input_channels(
         all_credentials = {}
 
     if channel:
+        if len(all_credentials) > 1:
+            logger.warning(
+                "Only credentials for chosen {} connector will be loaded.".format(
```

Comment:
I'd add some more explanation why this is happening. Maybe
`Connecting to channel '{}' which was specified by the '--connector' argument. Any other channels will be ignored. To connect to all given channels, omit the '--connector' argument.`